### DISCUSSION TITLE
Custom Piano Music: Megalovania

### DISCUSSION THREAD

**Discussion by FluffyMuffinMeyer** [Monika-After-Story/MonikaModDev#7787] [general - piano notes]
[Megalovania.txt](https://github.com/Monika-After-Story/MonikaModDev/files/2715734/Megalovania.txt)

Second Custom piano song. Lot's more lines than the Tetris theme.

Download, rename the file to Megalovania.json, make a folder in your DDLC Directory, name it "piano_songs", and place the Megalovania.json inside of that folder, if you then use the piano and play the song, it will appear under songs you can play anytime.

Just like the Tetris theme, the dialogs up in the air if anyone wants to change it, I wouldn't be surprised if I made some fixable mistakes, If you find any, leave a comment on this issue and i'll fix them as soon as possible.

Here are the Notes: (This ones a big one)
W W P U 7 Y T E T Y
W W P U 7 Y T E T Y
2 2 P U 7 Y T E T Y
Q Q P U 7 Y T E T Y
W W P U 7 Y T E T Y
W W P U 7 Y T E T Y
2 2 P U 7 Y T E T Y
Q Q P U 7 Y T E T Y
I I I I O 0 O I Y I O
I I I I O 0 P [ P ] ] ] P ] [
I I I I I I Y Y I I I I I Y I [ O I
U I O Y I O ] O I U 4 Y I - I P ]
I Y I O 0 O I Y 0 O I O
0 P ] P 0 O I Y U I O P ] ] 0 0 O I O
T Y U U Y T Y U I U P
P 0 O 9 I U 7 Y 6 7
I Y I O 0 O I Y 0 O I O
0 P ] P 0 O I Y U I O P ] ] 0 0 O I O
T Y U U Y T Y U I U P
P 0 O 9 I U 7 Y 6 7
Q 4 3 2 4
Q 4 3 2 2
W W P U 7 Y T E T Y
2 2 P U 7 Y T E T Y

(And no, I'm not sorry for the jokes.)
This song was as per request; https://github.com/Monika-After-Story/MonikaModDev/issues/3259

**Comment by ThePotatoGuy** [comment]
[keeping this open to remind me]

**Comment by pleasurepleader** [comment]
I put in the folder but Monika doesn't recognize it:(

**Comment by ThePotatoGuy** [comment]
upload `log/pnm.txt`

**Comment by pleasurepleader** [comment]
Alright, but how can I do it?
Sorry, I don't know😅

**Comment by multimokia** [comment]
Drag it into the message box here

**Comment by pleasurepleader** [comment]
Ok, thank you! I'll do it

**Comment by pleasurepleader** [comment]
[pnm.txt](https://github.com/Monika-After-Story/MonikaModDev/files/3023529/pnm.txt)

Here it is guys.

**Comment by pleasurepleader** [comment]
So What can I do?

**Comment by multimokia** [comment]
Looks like it's loaded in fine. Are you trying to play it through the `Play a song` menu? Or just free play?

**Comment by pleasurepleader** [comment]
I'm trying to play it in "Play a Song" but It doesn't appear there

**Comment by FluffyMuffinMeyer** [comment]
Sorry for not responding earlier. You have to successfully play the song once for it to appear in the "Play a Song" menu. Play the song normally, the text should appear of her referencing the song, that's how you know it will be working. Once you complete it, (It doesn't need to be perfect, just completed) she will add it too the "Play a Song" list.

**Comment by pleasurepleader** [comment]
Thank you so much, i'll do it

**Comment by pleasurepleader** [comment]
Yesterday I was doing it and it was working, she was saying: Wha-
But today she doesn't say something😅

**Comment by FluffyMuffinMeyer** [comment]
Make sure you are playing the notes correctly. If you mess up, or are to slow, she won't say anything.

**Comment by pleasurepleader** [comment]
Ok, I'll try it again
Thank you for your help btw😁

**Comment by pleasurepleader** [comment]
It is working, thank youuu!

**Comment by caliburn1337** [comment]
i want to change the notes, but idk what is the note for A S and /

**Comment by briaryoung** [comment]
There's a directory somewhere - I'm writing some code for a new song atm, but I can't find the directory so I'm doing it blindly :")

**Comment by briaryoung** [comment]
Also, I think it would be super funny if while you played this there was an artwork that referenced undertale- Ahahah! Sorry. I would love to see that while you played Meglovania for Monika- Probably out of character, though.

**Comment by FluffyMuffinMeyer** [comment]
@kkrosie123 @caliburn1337 We have a Wiki page dedicated for this. I used it extensively, it's a big help.
https://github.com/Monika-After-Story/MonikaModDev/wiki/Adding-Custom-Piano-Songs

**Comment by briaryoung** [comment]
Awesome, thank you!

**Comment by TheElementalOfDestruction** [comment]
Spelling error on line 249: "wasent" should be "Wasn't"

**Comment by TheCheddarCheese** [comment]
i'm a bit confused on the tempo after the first 4 lines, could i get some sort of video of how it's supposed to be played?
